cli/generators/crud/templates/components/list.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import <%= ucName %> from './<%= ucName %>';
+
+const <%= ucName %>List = ({ <%= pluralizedName %>, onClickDetail, onClickDelete }) => {
+  let <%= pluralizedName %>View = <p>Sorry, there are no <%= pluralizedName %> to show. You can try to add one.</p>;
+  if (<%= pluralizedName %>.length > 0) {
+    <%= pluralizedName %>View = <%= pluralizedName %>.map((<%= name %>) =>
+      <<%= ucName %>
+        key={<%= name %>.id}
+        id={<%= name %>.id}
+        name={<%= name %>.name}
+        onClickDetail={onClickDetail}
+        onClickDelete={onClickDelete} />
+    );
+  }
+  return (
+    <div>
+      {<%= pluralizedName %>View}
```

Comment:
You're generating the string "CarsView".

I'm suggesting you generate "Cars", since it's probably closer to the natural usage that the application will actually use.